main.tf

```diff
@@ -12,7 +12,7 @@
  * * [RedShift](https://aws.amazon.com/redshift/)
```

Comment:
Since this is already a breaking change, I recommend adding something to the top of these doc comments to explicitly call out whether this module is compatible with Terraform 0.11 or 0.12.

Reply:
Great idea. I just checked and after #23 gets merged in this should be compatible with both.

Reply:
approved that ;)

Reply:
Excellent I'll wait to tag until we get both in.